.github/workflows/workflow.yaml

```diff
@@ -104,8 +104,7 @@ jobs:
           if [[ $exists == null ]]
           then
             make docker-login
-            make docker-build IMG=gatekeeper-e2e:latest
-            make docker-push-dev IMG=gatekeeper-e2e:latest DEV_TAG=${GITHUB_SHA::7}
+            docker-buildx-dev IMG=gatekeeper-e2e:latest DEV_TAG=${GITHUB_SHA::7}
```

Comment:
passing in `IMG` doesn't apply anymore

Reply:
Just confirming, `DEV_TAG` is still required though?

Reply:
This would also be true for `VERSION` in the release